addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -1792,10 +1742,6 @@ export class Order extends PosModel {
         return true;
     }
 
-    isFirstDraft() {
```

Comment:
I think you can also remove all occurence of firstDraft in the code

Reply:
:+1: